simplq/src/components/common/Button/Button.jsx

```diff
@@ -4,11 +4,7 @@ import styles from '../../../styles/buttons.module.scss';
 const StandardButton = (props) => {
   const { onClick } = props;
   return (
-    <button
-      type="submit"
-      onClick={onClick}
-      className={props.outlined ? styles['standard-button-outlined'] : styles['standard-button']}
-    >
+    <button type="submit" onClick={onClick} className={styles['standard-button']}>
       {props.children}
```

Comment:
We still need the outlined buttons for certain pages like for the share and refresh queue buttons, so these shouldn't be removed. The same effect should be added to the 'standard-button-outlined' class as well, with a different transition color. Since this class extends the 'standard-button' class, just adding the different color should be enough.

Reply:
okay, cool